src/lib/ledger/mv3/fg.ts

```diff
@@ -0,0 +1,44 @@
+import browser from 'webextension-polyfill';
```

Comment:
the same https://github.com/madfish-solutions/templewallet-extension/pull/778/files#r1009236099